test/test.js

```diff
@@ -65,6 +65,24 @@ describe('static methods', function () {
     });
   });
 
+  describe('#fromString()', function () {
+    var originalVec, vec;
+
+    before(function () {
+      originalVec = new Vector(100, 200);
```

Comment:
Can you make one of these values more floaty? To make sure that `parseFloat` works.